CHANGELOG.rst

```diff
@@ -0,0 +1,4 @@
+:: 0.6
+
+* Updated tests, incl. complete API mocking
+* Updated test dependencies
```

Comment:
This file should be merged with the existing file HISTORY.rst

Reply:
@tcmetzger Thank you for catching that. I'll update it.

Reply:
Thanks!